src/D0_modelling_and_optimization.py

```diff
@@ -214,18 +214,21 @@ def plot_networkx_graph(dict_values, model):
             dict_values[SIMULATION_SETTINGS][DISPLAY_NX_GRAPH][VALUE] is True
             or dict_values[SIMULATION_SETTINGS][STORE_NX_GRAPH][VALUE] is True
```

Comment:
I would remove the setting `DISPLAY_NX_GRAPH`, as that was only needed because zooming in to the nx graph sometimes one could understand what the connections where better then in the png. with your new function, that is not needed anymore. 

Regarding `STORE_NX_GRAPH`: The name could be changed. If we wanted to minimize possible settings, we could simplify that it is always plotted then `-png` is used, and in the report for  `-pdf`. Though, this plot should be plotted where it is right now (not only after processing in F0), because it can solve some oemof-solph termination errors to look at it.

Reply:
I will tackle this in a new PR